src/controller/createQuestionnaireController.js

```diff
@@ -29,10 +30,12 @@ const createQuestionnaireController = async (req, res) => {
 
       const newQuestionnaire = await createQuestionnaire(Questionnaire, body);
       if(newQuestionnaire.errors){
+        logger.info('Questionnaire not created due to incorrect data')
         res.status(400).json(newQuestionnaire);
       }
       else {
         await newQuestionnaire.save();
+        logger.info('Questionnaire created')
```

Comment:
You can add the time that took to answer the request in all the response's logs. `process.hrtime.bigint()` can be useful for that purpose